pipenv/shells.py

```diff
@@ -36,19 +36,19 @@ def _get_activate_script(cmd, venv):
     does not work elsewhere anyway.
     """
     # Suffix and source command for various shells.
-    if cmd.endswith("/sh", "/bash", "/zsh"):
+    if cmd.endswith(("sh", "bash", "zsh")):
         suffix = ""
         command = "."
-    elif cmd.endswith("/fish"):
+    elif cmd.endswith("fish"):
```

Comment:
Shouldn't this come before the first conditional since fish ends in sh?

Reply:
True! Good catch. The order is a little crazy. Maybe `endswith` is the wrong way .
Pattern matching or Regex might be better?